packages/js/src/general/initialize.js

```diff
@@ -56,17 +60,34 @@ domReady( () => {
 	const endpoints = {
 		seoScores: get( window, "wpseoScriptData.dashboard.endpoints.seoScores", "" ),
 		readabilityScores: get( window, "wpseoScriptData.dashboard.endpoints.readabilityScores", "" ),
+		topPages: get( window, "wpseoScriptData.dashboard.endpoints.topPageResults", "" ),
 	};
 	/** @type {Object<string,string>} */
 	const headers = {
 		"X-Wp-Nonce": get( window, "wpseoScriptData.dashboard.nonce", "" ),
 	};
 
-	/** @type {{dashboardLearnMore: string}} */
+	/** @type {Links} */
 	const links = {
 		dashboardLearnMore: select( STORE_NAME ).selectLink( "https://yoa.st/dashboard-learn-more" ),
+		errorSupport: select( STORE_NAME ).selectAdminLink( "?page=wpseo_page_support" ),
 	};
 
+	const remoteDataProvider = new RemoteDataProvider( { headers } );
+	const dataProvider = new DataProvider( { contentTypes, userName, features, endpoints, headers, links } );
+	const widgetFactory = new WidgetFactory( dataProvider, remoteDataProvider );
+
+	const initialWidgets = [ "seoScores", "readabilityScores" ];
+	// If site kit feature is enabled, active and connected: add the top pages widget.
+	if (
+		get( window, "wpseoScriptData.dashboard.siteKitConfiguration.feature_enabled", false ) &&
+		get( window, "wpseoScriptData.dashboard.siteKitConfiguration.isActive", false ) &&
+		// get( window, "wpseoScriptData.dashboard.siteKitConfiguration.isConnected", false )
```

Comment:
Needs adaptation after: https://github.com/Yoast/wordpress-seo/pull/21978/commits/d7065dd6da8e1bc52f7448915ecb81b3b97d308e is merged

Reply:
Done